### DISCUSSION TITLE
Built SDRPP for Raspian Bookworm successfully,  SSTV and Meteor plugins  missing some header files.

### DISCUSSION THREAD

**Discussion by jerrynsr** [AlexandreRouma/SDRPlusPlus#1272] [Q&A]
Hi,

    I successfully built sdrpp for Raspian Bookworm, no issues.   I am having an issue building the SSTV and Meteor plugins.
I am missing the dsp/demodulator.h header file.   I checked the github tree and I can not find it there either.  Is this because SSTV and Meteor are in the process of being rewritten  or can you upload a fix to github ?

**Comment by gm8arv** [comment]
I would be interested in a bookworm version, even missing those plugins...

**Comment by jerrynsr** [comment]
I have SDR++ built on my RPI5, installed in a RASPAD 3.   RASPAD 3 isn't the best.  I think I'm going to move the build over to a 3d printed box, that has a 3w audio amplifier, decent speakers, and a HDMI 1080p 10" touch screen.   I did test SDR++ on the RPI5 using a RTL_SDR dongle and my HackRF.  I'll work on building a .deb package for it and I'll check with the maintainer to see if I can get access to upload it to Github.   When I'm home I'll boot up the Raspad 3 and work on building the .deb for Bookworm.
    On Tuesday, February 6, 2024 at 06:49:24 AM EST, David ***@***.***> wrote:  
 
 


I would be interested in a bookworm version, even missing those plugins...

**Reply by gm8arv** [reply]
Thanks, David.

**Comment by jerrynsr** [comment]
Hi David,             You can find the .deb file for Bookworm here:
https://github.com/jerrynsr/SDRPlusPlus/tree/patch-1.     I'll clean up my Github forked repos from AlexandreRouma/SDRPlusPlus as soon as my deb file is approved in the binary section.  I'm still working on building the wefax plugin.  I have some missing dependencies that do not exist in the AARCH64 Debian Bookworm repo.  I'm going to need to build them.  
I was listening to shortwave with this.    On Tuesday, February 6, 2024 at 08:50:24 AM EST, David ***@***.***> wrote:  
 
 


Thanks, David.

**Reply by gm8arv** [reply]
Thanks for that, Jerry.  After downloading I realized that was for Intel, and I needed armhf for the Raspberry Pi.  So I tried compiling myself but ended up with a 32-bit program despite all compiles being on a 64-bit system.  No idea where it picked up  32-bit from, or how to force it into building a 64-bit version.
